Shared/App/AppDelegate.swift

```diff
@@ -67,27 +67,32 @@ class AppDelegate: UIResponder, UIApplicationDelegate {
             completionHandler(.noData)
         }
     }
-    #endif
+#endif
+
 
     private func prepareCurrentUser() {
-        #if !NOTIFICATION
+#if !NOTIFICATION
         UserNotificationManager.shared.resetBadgeCount()
-        #endif
+#endif
 
-        #if !APPCLIP && !NOTIFICATION
+#if !APPCLIP && !NOTIFICATION
 
         guard !ChatClientManager.shared.isConnected else { return }
 
-        GetChatToken()
-            .makeRequest(andUpdate: [], viewsToIgnore: [])
-            .mainSink(receiveValue: { (token) in
+        Task {
+            do {
+                let token = try await GetChatToken().makeAsyncRequest(andUpdate: [], viewsToIgnore: [])
+
                 if ChatClientManager.shared.client.isNil {
                     ChatClientManager.shared.initialize(token: token)
                 } else {
                     ChatClientManager.shared.update(token: token)
                 }
-            }).store(in: &self.cancellables)
-        #endif
+            } catch {
+                print("===== \(error)")
+            }
+        }
```

Comment:
GetChatToken is the only cloud function actually using async in this PR. To call it from a non async context, we need to wrap it in a Task. It's also wrapped in a "do/catch" block for error handling.